frontend/dialob-composer-material/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "vite",
     "build": "jest && tsc && vite build",
+    "build:items": "tsc && vite build --config vite.config.items.ts",
```

Comment:
let's rename the target to 'lib', as this will be the "library" version of the composer. (so `build:lib`)